assets/style.css

```diff
@@ -0,0 +1,88 @@
+*{
+    box-sizing: border-box;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+header{
+    margin:0 0 10px 0;
+    padding: 10px 0;
+    height: 20px;
+}
+
+header img{
+    max-height:20px;
+}
+
+.video-block{
```

Comment:
You may consider using the more modern Flexbox layout instead of floats to take advantage of more streamlined and easier-to-implement responsive code, as well as smoother transitions across screen resolutions.